docs/html/index.rst

```diff
@@ -17,7 +17,7 @@ Please take a look at our documentation for how to install and use pip:
    ux_research_design
    news
 
-In 2020, we're working on improvements to the heart of pip: :ref:`Resolver changes 2020`. Please `learn more and take our survey`_ to help us do it right, and `join our user experience surveys pool`_.
+In pip 20.3, we're making a big improvement to the heart of pip: :ref:`Resolver changes 2020`. Please `learn more and take our survey`_ to help us do it right, and `join our user experience surveys pool`_.
```

Comment:
`learn more and take our survey`_

I'm unclear what survey you are talking about here. 

If it's the survey contained in the text "tell us whether it breaks anything for your setup by filling out this survey", then this link should link either a) directly to the survey, or b) directly to "Please make time to test the new version of pip, probably in May (see below)." via an achor.